themes/base.theme.bash

```diff
@@ -8,9 +8,20 @@ SCM_THEME_PROMPT_DIRTY=' ✗'
 SCM_THEME_PROMPT_CLEAN=' ✓'
 SCM_THEME_PROMPT_PREFIX=' |'
 SCM_THEME_PROMPT_SUFFIX='|'
+SCM_THEME_BRANCH_PREFIX=''
+SCM_THEME_TAG_PREFIX='tag:'
+SCM_THEME_COMMIT_PREFIX='commit:'
+SCM_THEME_REMOTE_PREFIX=''
+
+SCM_GIT_SHOW_DETAILS=${SCM_GIT_SHOW_DETAILS:=false}
```

Comment:
The only optional information is the working copy status.

Always shown information:

- Branch/Commit/Tag
- Commits ahead
- Commits behind
- Stashed changes

Optional ( visible setting SCM_GIT_SHOW_DETAILS=true):

- Staged files count
- Unstaged files count
- Untracked files count

 anyway, I will change the default value to true.